docs/fern/docs/pages/sdk/stack-app.mdx

```diff
@@ -545,19 +545,52 @@ console.log(user); // null if not found
 
 ### `listUsers()`
 
-List all users.
+List users with optional filtering and sorting.
+
+**Parameters:**
+
+<div className="indented">
+  <ParamField path="options" type="object">
+    <div className="indented">
+      <ParamField path="cursor" type="string">
+        The cursor to start the result set from.
+      </ParamField>
+
+      <ParamField path="limit" type="number" default="100">
+        The maximum number of items to return. Defaults to 100, max is 200.
```

Comment:
we have to talk about this. by default, `listUsers` should return all users (possibly making many API requests in the process), not just the first 100